criu/Makefile

```diff
@@ -22,6 +22,7 @@ CONFIG-DEFINES		+= -DUSER_CONFIG_DIR='".criu/"'
 #
 # General flags.
 CFLAGS			+= -fno-strict-aliasing
+CFLAGS			+= -fno-reorder-blocks-and-partition
```

Comment:
in the commit message you need to explain a reason why we need to this: why it was not a problem before your changes, when a problem was introduced and etc.

Reply:
> Most of them failed because when compiling, the file "criu/pie/restore-blob.h" would have a line `#define restorer_sym__export_restore_task.cold 0x475b`. I don't know why this happened cause the file was auto-generated. What I've done is changing some codes in `criu/pie/restorer.c`, function `__export_restore_task` . When I test the codes, using gcc 5.4.0 wouldn't produce this problem.

I'm not quite sure how "criu/pie/restore-blob.h" and the .cold symbol is generated. Also, using clang to compile wouldn’t produce this problem. It might be because I added some variables considering that using gcc 9.3.0 with original branch wouldn’t produce this problem. Using the flag `-fno-reorder-blocks-and-partition` prevent gcc from reordering basic blocks and partition into hot and cold sections so that the `.cold` symbol would disappear. Is there any way to generating `.cold` symbol? Or do you know why the `.cold` symbol generate?